tools/trap/HowToUseTrap.md

```diff
@@ -280,7 +280,7 @@ cd $TIZENRT_BASEDIR/tools/trap/
     $ python3 ramdumpParser.py -e `<Elf path>` -t `<Log file path>`
 
     ex)
-    $ python3 ramdumpParser.py -e ../../build/output/bin/app2_dbg -t ./log_file
+    $ python3 ramdumpParser.py -e ../../build/output/bin/tinyara -t ./log_file
```

Comment:
Regardless of crash point, can we use `tinyara` only?
If it is, why do we need to pass only this?

Reply:
Yes only tinyara can be used.
For app crash point detection we calculate app*_dbg based on the applications loaded for the configuration. (Line 592)

Reply:
Then, why do we need to pass -e option even app can be done automatically?

Reply:
If we need to pass the pass of bin folder for the location of elfs, then how about removing `tinyara` like `-e ../../build/output/bin`? If we can do that, then we had better change the option from -e to -l (?).

Reply:
Only apps have an elf naming convention like <app_name>_dbg
For kernel elf varies > tinyara/tinyara.axf 
Hence using this option.

Reply:
Kernel also has naming convention. Please find `EXEEXT` in os/Make.defs.
naming convention of tinyara
https://github.com/Samsung/TizenRT/blob/master/os/Makefile.unix#L228

Reply:
Only rtl8721csm has the EXEEXT value as axf so that tinyara.axf is used. Other boards has nothing for that. That's why tinyara is used for other boards.

Reply:
Even we change not to pass kernel elf, we need to pass the location because we can't guarantee this script can be executed at the same location.
Could you let me know your opinion for giving kernel elf and for giving location by user?